chore(project management): Pull Request Template (#491)
revoltchat/revite: .github/pull_request_template.md

## Please make sure to check the following tasks before opening and submitting a PR

> * [ ] I understood and followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)

Reviewer: Tense change:
```suggestion
* [ ] I understand and have followed the [contribution guide](https://github.com/revoltchat/revolt/discussions/282)
```

> * [ ] I tested my change locally and it is working as intended

Reviewer: Tense change:
```suggestion
* [ ] I have tested my change locally and it is working as intended
```

> * [ ] I have tested my change locally and it is working as intended

Reviewer: ```suggestion
* [ ] I have tested my changes locally and they are working as intended
```

> * [ ] The change doesn't have any notable side effects on other project

Reviewer: Minor typo:
```suggestion
* [ ] The change doesn't have any notable side effects on other Revolt projects
```

> * [ ] The change doesn't have any notable side effects on other Revolt projects

Reviewer: ```suggestion
* [ ] These changes do not have any notable side effects on other Revolt projects
```

> * [ ] (optional) I opened a pull request on the [translation repository](https://github.com/revoltchat/translations)

Reviewer: Tense chnage:
```suggestion
* [ ] (optional) I have opened a pull request on [the translation repository](https://github.com/revoltchat/translations)
```
